src/constants.ts

```diff
@@ -24,10 +24,7 @@ export const STATUS_PENDING = 'Pending'
 export const STATUS_INACTIVE = 'Inactive'
 export const STATUS_CANCELLED = 'Cancelled'
 
-export const SOURCE_PLANETSCOPE = 'planetscope'
-export const SOURCE_RAPIDEYE = 'rapideye'
-export const SOURCE_LANDSAT = 'landsat'
-export const SOURCE_SENTINEL = 'sentinel'
```

Comment:
We don't need these here anymore because they're stored in the SCENE_TILE_PROVIDERS right?

Reply:
Exactly.  They were only used to list the sources, which is now taken care of elsewhere.  The exception is `SOURCE_RAPIDEYE`, which was also used as the default source in search criteria.  I've replaced that with `SOURCE_DEFAULT`.